install_config/deploying_cfme.adoc

```diff
@@ -0,0 +1,510 @@
+functionally complete, and Red Hat does not recommend to use them for
+production. These features provide early access to upcoming product features,
+enabling customers to test functionality and provide feedback during the
+development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+The {product-title} installer includes an Ansible role and playbook for
+deploying Red Hat CloudForms 4.5 (CloudForms Management Engine 5.8, or CFME) on
+{product-title}. This deployment must be run on a dedicated {product-title}
+cluster, meaning that no applications or infrastructure other than those
+required for the CFME deployment should be running on it.
+
+This role is based on the work in the upstream
+link:https://github.com/ManageIQ/manageiq-pods[*manageiq/manageiq-pods*]
+project. For additional details on configurations specific to ManageIQ (MIQ),
+such as optional post-installation tasks, see the project's
+link:http://manageiq.org/docs/get-started/basic-configuration[upstream documentation].
+
+[NOTE]
+====
+The terms CFME and MIQ are interchangeable in this deployment.
+====
+
+[[deploying-cfme-requirements]]
+== Requirements
+
+[[deploying-cfme-prerequisites]]
+=== Prerequisites
+
+- A running {product-title} 3.6 cluster
+- NFS or other compatible persistent volume storage provider
+- A *cluster-admin* user (created by this role, if required)
+
+[[deploying-cfme-cluster-sizing]]
+=== Cluster Sizing
+
+In order to avoid random deployment failures due to insufficient resources, the
+following minimum cluster size is recommended for a test environment:
+
+[cols="4*", options="header"]
+|===
+|Type
+|Size
+|CPUs
+|Memory
+
+|Masters
+|1+
+|8
+|12 GB
+
+|Nodes
+|2+
+|4
+|8 GB
+
+|Persistent Volume (PV) Storage
+|25 GB
+|N/A
+|N/A
+|===
+
+[IMPORTANT]
+====
+CFME has hard requirements for memory. CFME will not install if your
+infrastructure does not meet or exceed the requirements given above.
+====
+
+[[deploying-cfme-other-sizing-considerations]]
+=== Other Sizing Considerations
+
+- Recommendations assume CFME will be the only application running on this {product-title} cluster.
+- Alternatively, you can provision an infrastructure node to run registry, router, metrics, and/or logging pods.
+- Each CFME application pod will consume at least 3 GB of RAM on initial deployment (a blank deployment without providers).
+- RAM consumption will ramp up higher depending on appliance use; after providers are added, expect higher resource consumption.
+
+[[deploying-cfme-assumptions]]
+=== Assumptions
+
+. You must meet or exceed the xref:deploying-cfme-cluster-sizing[Cluster Sizing] requirements.
+. Your NFS server is on your master host.
+. Your PV backing the NFS storage volume is mounted on *_/exports/_*.
+
+The following directories that NFS will export to back the PVs are required:
```

Comment:
Is the above missing a word? Maybe:

"The following are the directories that are required by NFS, so it can export files in order to back the PVs"

Is it "back" that's incorrect? My suggestions doesn't seem very correct...